sci-astronomy/stellarium/stellarium-25.1.recipe

```diff
@@ -7,11 +7,11 @@ COPYRIGHT="2000-2024 Stellarium Developers"
 LICENSE="GNU GPL v2"
 REVISION="1"
 SOURCE_URI="https://github.com/Stellarium/stellarium/releases/download/v$portVersion/stellarium-$portVersion.tar.gz"
-CHECKSUM_SHA256="433bb8a0380313e6d9e4caf62395b724f4d74262c8aae426a0fe1a5e61a29cbb"
+CHECKSUM_SHA256="2ecbc90f0bc7f1355e4d6497db2adc9119000d88eb18a3b4257ce249c4bea2f0"
 ADDITIONAL_FILES="stellarium.rdef.in"
 
 ARCHITECTURES="all !x86_gcc2"
-SECONDARY_ARCHITECTURES="x86"
+SECONDARY_ARCHITECTURES="!all"
```

Comment:
what is meant with this?

Reply:
Well, a quick solution for disabling building x86 package. Probably removing line _SECONDARY_ARCHITECTURES_ and _$secondaryArchSuffix_ is better solution in ideological plan for packaging only for 64-bit architectures…

Reply:
Is the x86 build broken?

Reply:
> Is the x86 build broken?

Builder has no enough resources for compiling the package. In any way the newest star catalogs using 64-bit identifiers for stars and we (stellarium team) stopped preparation an official 32-bits binaries.